packages/docusaurus/src/server/translations/translationsExtractor.ts

```diff
@@ -210,12 +210,12 @@ function extractSourceCodeAstTranslations(
         path.node.openingElement.name.name === 'Translate'
       ) {
         // We only handle the optimistic case where we have a single non-empty content
-        const singleChildren: NodePath | undefined = path
```

Comment:
After bump of `@babel/core`, apparently type `NodePath` has been "extended", so I removed its using in two places so that the build succeeds again.

Reply:
looks fine to me.

I think I added the types because TS complained otherwise so if it still compiles... ;)